rejson.go

```diff
@@ -1,510 +1,310 @@
 package rejson
 
 import (
-	"encoding/json"
-	"fmt"
-	"strings"
-
-	"github.com/gomodule/redigo/redis"
-)
-
-const (
-	// PopArrLast gives index of the last element for JSONArrPop
-	PopArrLast = -1
-	// DebugMemorySubcommand provide the corresponding MEMORY sub commands for JSONDebug
-	DebugMemorySubcommand = "MEMORY"
-	// DebugHelpSubcommand provide the corresponding HELP sub commands for JSONDebug
-	DebugHelpSubcommand = "HELP"
-	// DebugHelpOutput is the output of command JSON.Debug HELP <obj> [path]
-	DebugHelpOutput = "MEMORY <key> [path] - reports memory usage\nHELP                - this message"
+	"github.com/Shivam010/go-rejson/rjs"
```

Comment:
This will need to be changed : s/Shivam010/niitshm/ before pushing

Reply:
Don't worry about that.